cloud/api-gateway/src/main/resources/application.yaml

```diff
@@ -1,3 +1,31 @@
 spring:
   application:
-    name: api-gateway
\ No newline at end of file
+    name: api-gateway
+  cloud:
+      gateway:
+        routes:
+          - id: user-manager
+            uri: http://localhost:8081
+            predicates:
+              - Path=/api/v1/customers/**,/api/v1/merchants/**
+          - id: payment-manager
+            uri: http://localhost:8082
+            predicates:
+              - Path=/api/v1/wallets/**,/api/v1/payments/**
+          - id: notification-manager
+            uri: http://localhost:8083
+            predicates:
+              - Path=/api/v1/notifications/**
+          - id: qr-generator
+            uri: http://localhost:8085
+            predicates:
+              - Path=/api/v1/qrcode/**
+          - id: auth-manager
+            uri: http://localhost:8086
+            predicates:
+              - Path=/api/v1/identity/users/**,/api/v1/auth/**
```

Comment:
Actually this path /api/v1/identity/users/** should not be in the API Gateway. This is for internal use only. We don't allow users directly send requests to authManager to create users. Same for notifications
Btw identity is not relevant name. Maybe `credentials` ?